assets/js/state/sagas/hosts.test.js

```diff
@@ -0,0 +1,18 @@
+import { recordSaga } from '@lib/test-utils';
+import { hostDeregistered } from '@state/sagas/hosts';
+import { HOST_DEREGISTERED, removeHost } from '@state/hosts';
+import { hostFactory } from '@lib/test-utils/factories';
+
+describe('Hosts sagas', () => {
+  it('should remove the host', async () => {
+    const { id, hostname } = hostFactory.build();
```

Comment:
This doesn't make much sense, right?
Why not simply do `const payload = hostFactory.build()` ?
Having more fields from the factory doesn't affect anything as far as I see

Reply:
The extra fields cause the check using `toContainEqual()` to fail. Trying to see if there's an easy way to get it to work in Jest.